implementation/revapi.json

```diff
@@ -0,0 +1,51 @@
+[
+  {
+    "extension": "revapi.java",
+    "id": "java",
+    "configuration": {
+      "missing-classes": {
+        "behavior": "report",
+        "ignoreMissingAnnotations": false
+      },
+      "filter": {
+        "packages": {
+          "regex": true,
+          "include": [
+            "io\\.smallrye\\.mutiny\\..*"
+          ],
+          "exclude": [
+            "io\\.smallrye\\.mutiny\\.operators\\..*"
+          ]
+        }
+      }
+    }
+  },
+
+  {
+    "extension": "revapi.differences",
+    "id": "breaking-changes",
+    "configuration": {
+      "criticality": "highlight",
+      "differences": [
```

Comment:
This has to be maintened after each release right? (to remove the older items)

Reply:
Yes, unfortunately...

Trying to see if I can find a way to "clear the array on release".